metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-letsencrypt",
-  "version": "2.2.1-rc0",
+  "version": "2.2.1-rc1",
```

Comment:
that isn't needed but also doesn't hurt anybody. Our release process will bump this to a non-rc version.